src/platform/packages/shared/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -94,6 +94,7 @@ const ALERT_GROUPING = `${ALERT_NAMESPACE}.grouping` as const;
 const ALERT_GROUP = `${ALERT_NAMESPACE}.group` as const;
 const ALERT_GROUP_FIELD = `${ALERT_GROUP}.field` as const;
 const ALERT_GROUP_VALUE = `${ALERT_GROUP}.value` as const;
+const ALERT_INDEX_PATTERN = `${ALERT_NAMESPACE}.index_pattern` as const;
```

Comment:
Looking at a [previous PR](https://github.com/elastic/kibana/pull/184163/files#diff-9645c0b91287093527bc56068a73530a47e3471beb54646f147ca4433b22f8e0), we added changes like this in `default_alerts_as_data.ts`, and not this file.  I'm not familar with what the difference might be though.

Reply:
After taking a quick look it seems like `default_alerts_as_data.ts` is the right file for this.

I've moved it there in this commit: https://github.com/elastic/kibana/pull/239450/commits/9a2fcd3e317b4ce4db3785693f04a522967fd4a1.

Let me know if I should revert it, thanks!